marlowe-runtime/doc/ReadMe.md

```diff
@@ -1,100 +1,141 @@
+Operating in concert and relying upon  : 
+- [cardano-node](https://github.com/input-output-hk/cardano-node/blob/master/README.rst) for blockchain connectivity,
+- [PostgreSQL](https://www.postgresql.org/) for persistent storage. 
 
-![Marlowe Runtime ecosystem](diagrams/ecosystem.png)
+Marlowe runtime functionalities are accessible via: 
+- Command-line client (`marlowe`), 
+- AWS-λ function (`marlowe-lambda`), 
+- REST/WebSocket server (`web-server`) that uses JSON payloads. 
 
+For transaction signing :
+ - Web applications can integrate with a [CIP-30 light wallet](https://cips.cardano.org/cips/cip30/)
+ - Enterprise applications can integrate with : 
+	- [cardano-wallet](https://github.com/input-output-hk/cardano-wallet/blob/master/README.md), 
+	- [cardano-cli](https://github.com/input-output-hk/cardano-node/blob/master/cardano-cli/README.md), 
+	- [cardano-hw-cli](https://github.com/vacuumlabs/cardano-hw-cli/blob/develop/README.md).
 
-## Backend Services
 
-The backend services for use typed protocols over TCP sockets, with ports separate by concerns (i.e., control, query, and synchronization). Each service handles rollbacks via the use of intersection points that reference specific slots/blocks on the blockchain. Most of the data flow is stream-oriented and the services are as stateless as possible. The information flow within the backend maximizes the node as the single source of truth, thus minimizing the danger of downstream components receiving inconsistent information. The Haskell types in the client API for Runtime Clients are decoupled from dependencies upon the numerous Cardano packages for the ledger, node, plutus, etc., so that a Haskell client for Runtime has very few Runtime or Cardano dependencies in its `.cabal` file.
+## Backend Services
 
-![Backend component services in Marlowe Runtime](diagrams/components.svg)
+The backend services use typed protocols over TCP sockets, with concerns separated by ports (i.e., control, query, and synchronization). 
 
-See the [tutorial for the Marlowe Runtime command-line-interface](tutorial.md) or the [deployment instructions for the Marlowe Runtime backend](deployment.md) for more information.
+Each service handles rollbacks via the use of intersection points that reference specific slots/blocks on the blockchain. 
 
+Most of the data flow is stream-oriented and the services are as stateless as possible. The information flow within the backend maximizes the node as the single source of truth, thus minimizing the danger of downstream components receiving inconsistent information. 
 
-### Chain Seek Daemon
+The Haskell types in the client API for Runtime Clients are decoupled from dependencies upon the numerous Cardano packages for the ledger, node, plutus, etc., so that a Haskell client for Runtime has very few Runtime or Cardano dependencies in its `.cabal` file.
```

Comment:
This isn't true, unfortunately (though I would certainly like for it to be true one day...)

Reply:
Currently, `cardano-api` `cardano-ledger-core`, `ouroboros-consensus`, `ouroboros-network`, `plutus-tx`, and `plutus-ledger-api` are all dependencies.

Reply:
replaced it with :
```
Runtime Clients currently depend on the following cardano packages : 
- cardano-api 
- cardano-ledger-core, 
- ouroboros-consensus, 
- ouroboros-network, 
- plutus-tx, 
- plutus-ledger-api 

Efforts are and will be made, so that a Haskell client for Runtime has very few Cardano dependencies in its `.cabal` file.
```